### PR TITLE
feat: redesign `guides/htpc.md`

### PR DESCRIPTION
**Pull request by ricky8k**
Redesign for `guides/htpc.md` following the new guide in https://github.com/Snaacky/thewiki/pull/160.

Changes include:
- Hosting images on `master` instead of PR fork (also, Discord's CDN makes you download the example of Kodi rendering subs incorrectly)
- Removing personal language. I believe that it makes the guide feel more like a blog and unfit with the rest of the wiki
- Adding more clarification to important details
- Amending grammatical/formatting errors

### CODE REVIEW

File: docs/guides/htpc.md [hunk 1]
```diff
@@ -1,95 +1,143 @@
...
+
+### GPU
 
-There are two main considerations for an Anime HTPC. The biggest, will be the CPU. You will need to decide if you want to use the integrated GPU (iGPU) or if you want to install an external dedicated GPU (dGPU).
+Generally, a dedicated graphics card (dGPU) will almost always be better than the integrated option (iGPU). *However, you may want to consider whether your home-theater setup will necessitate one if you're on a tighter budget:*
 
-+++ dGPU
++++ Dedicated
+
+A dGPU is a dedicated piece of hardware which will be used to handle video decoding. Cards in this category are usually from *AMD* or *NVIDIA*, with *Intel* potentially for newer codecs.
```
Comment (Ravencentric) [inline docs/guides/htpc.md:28]: > with *Intel* potentially for newer codecs.
Intel's only advantage was having AV1, but both nvidia and amd have it now with much better drivers than what intel offers. Intel is also new which means it's less bang for your bucks.

File: docs/guides/htpc.md [hunk 2]
```diff
@@ -119,23 +167,23 @@ vo=gpu-next
...
+## Above makes use of vo=gpu-next for HDR Content, and vo=gpu for SDR. This is a requirement for blend-subtitles=video
 ```
 
 ==-
 
-In my input.conf I have a single line, for deband.
+#### Scripts
 
-```yaml
-BACK cycle deband
```
Comment (Ravencentric) [inline docs/guides/htpc.md:132]: You should leave some hint about tying deband to a key

File: docs/guides/htpc.md (hunk 1 above)
Comment (ricky8k) [reply docs/guides/htpc.md:28]: Alright. Should we add a note instead since Intel makes dGPUs?

Comment (Ravencentric) [reply docs/guides/htpc.md:28]: They already killed their ARC line of GPUs, and currently only have LE (low performance) model. There's no gurantee if they will kill it or not so I think we can just ignore them for now

File: docs/guides/htpc.md (hunk 2 above)
Comment (ricky8k) [reply docs/guides/htpc.md:132]: Resolved in https://github.com/Snaacky/thewiki/pull/166/commits/833bbf18904ae7deee475422b3b6489cd05a32a9.

File: docs/guides/htpc.md (hunk 1 above)
Comment (ricky8k) [reply docs/guides/htpc.md]: Resolved in https://github.com/Snaacky/thewiki/pull/166/commits/833bbf18904ae7deee475422b3b6489cd05a32a9.

**Pull request merged**